content/redirections/to-get-your-app-token.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
why this filename? couldn't it be the `apps-getting-started.html`?

Reply:
Discussed IRL